var/spack/repos/builtin/packages/py-boost-histogram/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBoostHistogram(PythonPackage):
+    """The Boost::Histogram Python wrapper."""
+
+    homepage = "https://github.com/scikit-hep/boost-histogram"
+    pypi     = "boost_histogram/boost_histogram-1.2.1.tar.gz"
+
+    version('1.2.1', sha256='a27842b2f1cfecc509382da2b25b03056354696482b38ec3c0220af0fc9b7579')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@45:', type='build')
    depends_on('py-setuptools-scm@4.1.2:+toml', type='build')
```
These come from the `pyproject.toml`, which we currently don't use, but will use once #27798 is merged.